average.js

```diff
@@ -1,6 +1,14 @@
-
 function average(numbers) {
-  return numbers.reduce((p, c)=> p + c, 0) / numbers.length;
+  let totalNotNulls = numbers.length;
```

Comment:
Clear naming and code to capture the intent of the requirement :+1: 
The counting and summation is happening in the same loop, which is an advantage in many situation.
Consider the other side as well... doing them in two separate loops enables parallel execution, which may reduce the time taken in case of large datasets.